app/src/backend/hiap/ActionPlanService.ts

```diff
@@ -11,6 +11,8 @@ export interface CreateActionPlanInput {
   actionId: string;
   highImpactActionRankedId?: string;
   cityLocode: string;
+  cityId?: string;
+  inventoryId?: string;
```

Comment:
How do we create the values for the existing actions in e.g. the Brazil inventories?
Should we have an admin route or a script to assign them based on the cityName?
Or do we need to re-generate all of the actions?